app/controllers/topics_controller.rb

```diff
@@ -697,7 +697,7 @@ def destroy
     PostDestroyer.new(
       current_user,
       topic.ordered_posts.with_deleted.first,
-      context: params[:context],
+      context: params[:context] || I18n.t("staff_action_logs.api_post_delete"),
```

Comment:
Maybe we should do `params[:context].presence || …` just in case?

Reply:
Great point! 👍